h2/src/test/org/h2/test/scripts/testScript.sql

```diff
@@ -7103,3 +7103,48 @@ select * from test where "YEAR" in (select distinct "YEAR" from test order by "Y
 
 drop table test;
 > ok
+
+----- Issue#2390 -----
+CREATE TABLE TEST_TABLE ("ID" NUMBER(19, 0), "TEXT" VARCHAR(100));
```

Comment:
1. Please, don't add new tests to `testScript.sql` and `testSimple.sql`, use some files relevant to the command that you want to fix.
2. Use explicit `CREATE MEMORY TABLE` in tests of `SCRIPT` command to avoid failures during tests of persistent databases.
3. `NUMBER` is not a native data type of H2; it would be better to use `NUMERIC(19)` or any other standard or H2's own data type.